Capstone_EDA.ipynb

```diff
@@ -58,7 +58,7 @@
     "\n",
```

Comment:
<p>Ist es wirklich gut, diese beiden Teile (Daten aus csv lesen / Daten aus pkl lesen) in eine gemeinsame Zelle zu packen? Müssen wir dann nicht sehr häufig unterschiedliche Teile auskommentieren?</p><p>Vielleicht ein Denkfehler von mir...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/davidslabon/Capstone_UnlockingClimateSolutions/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Mein Gedanke ist, dass das (im Idealfall) überall nur 1x laufen muss. Wenn die Pickle dann bei euch sind laden wir nur noch die Pickle.</p><p>Dafür wird dann im Notebook überall das csv-lesen und Pkw-schreiben auskommentiert und wir lassen nur noch das pkl-lesen ausführen.</p><p>Das kann dann auch so bleiben... Aber lass uns das nachher vielleicht nochmal in der Runde besprechen...  </p>